supplementary_style_guide/style_guidelines/specific-doc-types.adoc

```diff
@@ -2,13 +2,42 @@
 [[specific-doc-types]]
 = Specific documentation types
 
+[[prerequisites]]
+== Prerequisites
+
+When writing prerequisites, be as clear and concise as possible. You can use the passive voice, _if necessary_, to achieve that end.
+
+Write prerequisites as checks that are true or that the user must have completed before they begin a procedure. They can be actions that the user, another person, or piece of technology has completed. Prerequisites can also include items that the user must have ready before beginning the procedure.
+
+* The passive voice might be appropriate for a prerequisite that is not completed by the current user. For example, having a configuration enabled by a system admin.
+
+* Avoid using imperative formations.
+
+* Use parallel language when writing prerequisites. For example, if one bullet is a complete sentence, the other bullets should be complete sentences. But one bullet could be passive voice and another active voice.
+
+.Examples of prerequisites
+
+* JDK 11 or later is installed.
++
+Passive voice: the agent is unknown or unimportant.
+
+* A running Kafka instance in {product}.
```

Comment:
I'd add this sentence:

++
+ Not a complete sentence: This prerequisite is acceptable if all the other prerequisites in your list are also not complete sentences.

Reply:
@stoobie With this small change, I'd go ahead and approve the pull request.